pkg/bbgo/modify.go

```diff
@@ -0,0 +1,74 @@
+package bbgo
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/c9s/bbgo/pkg/interact"
+	"github.com/c9s/bbgo/pkg/util"
+	log "github.com/sirupsen/logrus"
+)
+
+func RegisterModifier(s interface{}) {
+	val := reflect.ValueOf(s)
+	if val.Type().Kind() == util.Pointer {
+		val = val.Elem()
+	}
+	var targetName string
+	var currVal interface{}
+	var mapping map[string]string
+	// currently we only allow users to modify the first layer of fields
+	RegisterCommand("/modify", "Modify config", func(reply interact.Reply) {
+		reply.Message("Please choose the field name in config to modify:")
+		mapping = make(map[string]string)
+		for i := 0; i < val.Type().NumField(); i++ {
```

Comment:
can you pull out this to the dynamic.GetModifiableFields or dynamic.GetChangableFields function?

Reply:
done